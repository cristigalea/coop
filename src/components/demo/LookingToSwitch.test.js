import React from 'react';
import {shallow} from 'enzyme';

import LookingToSwitch from './LookingToSwitch';
import { StyleSheetTestUtils} from 'aphrodite';

describe('<LookingToSwitch />', () => {
    beforeEach(() => {
        StyleSheetTestUtils.suppressStyleInjection();
    });
      
    afterEach(() => {
        StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
    });

    it('should match the snapshot', () => {
        const component = shallow(<LookingToSwitch user='cristi' githubData={{}} getGithubInfo={() => {}} />);
        expect(component).toMatchSnapshot();
    });
});