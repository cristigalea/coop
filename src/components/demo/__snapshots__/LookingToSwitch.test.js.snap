// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LookingToSwitch /> should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LookingToSwitch
    getGithubInfo={[Function]}
    githubData={Object {}}
    user="cristi"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "No github info",
    },
    "ref": null,
    "rendered": "No github info",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "No github info",
      },
      "ref": null,
      "rendered": "No github info",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
